examples/recommendations/get_recommendation_impact_metrics.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example is to get impact metrics for a custom budget. 
+
+Use this example to get impact metrics for a given budget amount.
+
+This example uses the following: 
+1) Performance Max for the campaign type
+2) United States for the geo targeting 
+3) Maximize Conversions Value for the bidding strategy
+
+To get budget recommendations, run generate_budget_recommendations.py.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.googleads.client import GoogleAdsClient
+from google.ads.googleads.errors import GoogleAdsException
+
+
+def main(client, customer_id, user_provided_budget_amount):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+        user_provided_budget_amount: a user-provided budget amount (not in micros), to retrieve impact metrics for.
+    """
+    recommendation_service = client.get_service("RecommendationService")
+    request = client.get_type("GenerateRecommendationsRequest")
+
+    request.customer_id = customer_id
+    request.recommendation_types = ["CAMPAIGN_BUDGET"]
+    request.advertising_channel_type = client.enums.AdvertisingChannelTypeEnum.PERFORMANCE_MAX
+    request.bidding_info.bidding_strategy_type = "MAXIMIZE_CONVERSION_VALUE"
+    request.positive_locations_ids = [2840]  # 2840 is for United States
+    request.asset_group_info = [{ "final_url": "https://www.your-company.com/" }]
+    request.budget_info.current_budget = round((user_provided_budget_amount*1000000), 2)
```

Comment:
can you add a comment above this to say: "Multiply the user-provided budget by 1,000,000 to convert to micros, as required for current_budget."  (and have you confirmed it's meant to be in micros? The protos don't specify, and I think we'll want to add that in comments in v19_1 if that's the case).

Reply:
Great, thanks for confirming! I've filed a bug to update the comments and cc'd you.